### ISSUE TITLE
Weird background in macOS Sierra

### ISSUE THREAD

**Issue by sarbull** [atom/atom#15083]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Weird backround in macOS Sierra.

### Steps to Reproduce

1. Install Atom
2. Open with default "One Dark"
3. Open any file
4. Background is not fully one colored

**Expected behavior:** Same background color on all window

**Actual behavior:** Ununiform background pattern

**Reproduces how often:** Every time i open Atom

### Versions

`For macOS 10.8 or later`

### Additional Information

![image](https://user-images.githubusercontent.com/1864112/28541979-67d9e6e2-70c3-11e7-8a0c-c04d099a040a.png)

**Comment by rsese** [comment]
Thanks for taking the time to contribute!

We noticed that this is a duplicate of #12634. You may want to subscribe there for updates.

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).

**Comment by sarbull** [comment]
Hi,

8 September 2016, that's an oldie..

**Comment by rsese** [comment]
> 8 September 2016, that's an oldie..

Yeah it was waiting on an Electron upgrade and should be fixed in 1.19.0 /cc https://github.com/atom/atom/issues/12634#issuecomment-302892306.
